linetools/spectralline.py

```diff
@@ -126,8 +127,9 @@ def from_dict(cls, idict, coord=None, warn_only=False, chk_data=True, **kwargs):
                 #pdb.set_trace()
                 sline.analy[key] = ltu.convert_quantity_in_dict(idict['analy'][key])
             elif key == 'spec_file':
-                warnings.warn("You will need to load {:s} into attrib['spec'] yourself".format(
+                warnings.warn("You will need to load {:s} into analy['spec'] yourself".format(
                         idict['analy'][key]))
```

Comment:
Can you add a commented line explaining that analy['spec'] is expected to be `XSpectrum1D` and `analy['spec_file']` is the filename of the spectrum.